code/modules/vore/fluffstuff/custom_items_vr.dm

```diff
@@ -620,6 +620,139 @@
 	new /obj/item/weapon/paper/khcrystal_manual(src)
 	new /obj/item/clothing/accessory/collar/khcrystal(src)
 
+/obj/item/clothing/accessory/dosimeter
```

Comment:
This should probably be moved to a different file. custom_items_vr.dm is very much a bloated file and realistically should probably only contain fluff items. I recommend VOREStation\code\modules\clothing\under\accessories\badges_vr.dm (You will have to create a few file so make sure to add it to the dmi)

Reply:
Yeah, I noticed too that it seemed pretty bloated. The general idea was based around the life crystal so I figured it would be the right file.

I moved it to the badges_vr one now.